javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,10 @@
+/* exported filterOutStrings */
+function filterOutStrings(values) {
+  const newArr = [];
+  for (let i = 0; i < values.length; i++) {
+    if (typeof (values[i]) === 'number') {
+      newArr.push(values[i]);
+    }
```

Comment:
Here it would be safer to exclude strings (filter out strings) instead of add numbers.